Engine/Entities/Ants/Ant.cs

```diff
@@ -137,14 +139,18 @@ public bool PickUp(ResourceEntity e)
 
             return false;
         }
-
+        
         /// <summary>
         /// The Ant emits a home pheromone.
         /// </summary>
         public void EmitHomePheromone()
         {
             Transform homeTransform = new(Transform.Position, 0, Vector2.One);
-            HomePheromone unused = new(Name, homeTransform, World, PheromoneTimeSpan);
+
+            if (!ReinforceNearestPheromone<HomePheromone>())
+            {
+                HomePheromone unused = new(Name, homeTransform, World, PheromoneTimeSpan);
```

Comment:
What's the reasoning behind calling the variable unused?

Reply:
This is just because we don't use this newly created pheromone, we just want to call the constructor